processor-steps/src/main/java/boringyuri/processor/common/steps/AssociatedUriDataGeneratorStep.kt

```diff
@@ -13,59 +13,73 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package boringyuri.processor
+package boringyuri.processor.common.steps
 
-import boringyuri.api.DefaultValue
-import boringyuri.api.Param
-import boringyuri.api.Path
-import boringyuri.api.UriBuilder
-import boringyuri.api.WithUriData
+import androidx.room.compiler.processing.*
+import boringyuri.api.*
```

Comment:
@bomiyr could you please fix you codestyle rules not to aggregate imports and run import optiizer in this file to unfold them back.

Reply:
done